packages/manager/apps/public-cloud/src/assets/theme/_typography.less

```diff
@@ -6,3 +6,22 @@ h1 {
   color: @text-color;
   font-weight: 400;
 }
+
+h2 {
```

Comment:
Is applying theses changes to `.oui-heading_X` would be useful too?

Reply:
done